Source/Hurl.BrowserSelector/Windows/MainWindow.xaml

```diff
@@ -6,170 +6,133 @@
     xmlns:local="clr-namespace:Hurl.BrowserSelector.Windows"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     xmlns:tray="http://schemas.lepo.co/wpfui/2022/xaml/tray"
-    xmlns:vs="clr-namespace:Hurl.BrowserSelector.Controls"
+    xmlns:controls="clr-namespace:Hurl.BrowserSelector.Controls"
     xmlns:wpfui="http://schemas.lepo.co/wpfui/2022/xaml"
-    Title="Hurl - Select a Browser"
-    MinWidth="410"
-    MinHeight="220"
-    MaxWidth="560"
-    MaxHeight="450"
+    Title="Select a browser - Hurl"
+    MinWidth="480"
+    MinHeight="320"
     Deactivated="Window_Deactivated"
     ExtendsContentIntoTitleBar="True"
     PreviewKeyUp="Window_KeyEvents"
     SizeChanged="Window_SizeChanged"
     Topmost="True"
     WindowBackdropType="Mica"
     WindowStartupLocation="CenterScreen"
-    mc:Ignorable="d">
-    <Border
-        x:Name="WindowBorder"
-        BorderBrush="AliceBlue"
-        BorderThickness="2"
-        CornerRadius="8">
-        <Grid>
-            <Grid.RowDefinitions>
-                <RowDefinition Height="40" />
-                <RowDefinition Height="1*" />
-                <RowDefinition Height="45" />
-            </Grid.RowDefinitions>
+    mc:Ignorable="d" Icon="/Windows/internet.ico">
```

Comment:
How does adding Icon help here ? cause even this is not present, the icon is still visible

Reply:
@Jay-o-Way